### DISCUSSION TITLE
I need a Badge

### DISCUSSION THREAD

**Discussion by arad1252x** [community/community#135162] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

where can i get badges on github for about section

**Comment by everforyou** [comment]
Hi @arad1252x,

Websites like [Shields.io](https://shields.io/) allow you to create custom badges for your repositories. You can use these badges to showcase your skills, achievements, or project statuses.

**Reply by aradfarahani** [reply, marked as answer]
You can get badges for your GitHub profile's "About" section from several sources:

1. **Shields.io**: This is a popular site for creating customizable badges. You can generate badges for various metrics like build status, version, license, and more. Visit [Shields.io](https://shields.io/) to create your badges.

2. **GitHub Badge Generator**: This tool allows you to generate HTML for GitHub badges that you can copy and paste into your README file or profile. Check it out [here](https://badges.ritza.co/).

3. **Awesome Badges**: This is a curated list of GitHub badges for various purposes, including social media, skills, and more. You can find it on the [DEV Community](https://dev.to/envoy_/150-badges-for-github-pnk).


and dont forget to learn markdown its aLL about markdown. copy and paste it...
